modules/core/src/v2/coins/xlm.ts

```diff
@@ -42,8 +42,10 @@ import { checkKrsProvider, getStellarKeys } from '../recovery/initiate';
 const co = Bluebird.coroutine;
 
 interface AddressDetails {
+  baseAddress?: string;
   address: string;
```

Comment:
could you document the difference between `address` and `baseAddress`?

Reply:
done